Validation/Geometry/test/genTrackerPlots.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+VGEO_DIR=$CMSSW_BASE/src/Validation/Geometry
+TEST_DIR=${VGEO_DIR}/test/materialBudgetTrackerPlots
+
+if [ ! -d $TEST_DIR ]; then
+    mkdir $TEST_DIR && cd $TEST_DIR
+else
+    rm -rf $TEST_DIR && cd $TEST_DIR
```

Comment:
this will fail , you need to create the directory again before `cd` in to it. I would suggest to just have replace this with
```
if [ -d $TEST_DIR ] ; then rm -rf $TEST_DIR ; fi
mkdir $TEST_DIR && cd $TEST_DIR
```

Reply:
Thanks for pointing out. Resolved with last commit